docs/configuration/production.rst

```diff
@@ -76,24 +108,34 @@ change the default privileges to allow reading the future tables:
     ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT SELECT ON TABLES TO dbuser;
 
 
-Initialization
---------------
+Creating a configuration file
+-----------------------------
 
-Once a PostgreSQL is up and running somewhere, select the PostgreSQL option when
-running the ``init`` command:
+Once a PostgreSQL is up and running somewhere, you should edit your configuration file or create one with the ``init`` command and select the PostgreSQL option. The file is created by default as ``config\kinto.ini``, to specify another name use the ``--ini`` parameter.
 
-.. code-block :: bash
+.. code-block:: bash
 
     $ kinto --ini production.ini init
 
-By default, the generated configuration refers to a ``postgres`` database on
-``localhost:5432``, with user/password ``postgres``/``postgres``. If you want
-to change that, make sure to update the :ref:`backends setting <configuration-backends>`
-(eg: ``postgres://myuser:mypass@localhost:5432/mydb``).
+
+Select your database server address, name and user by editing the configuration file. Also make sure that the PostgreSQL :ref:`backend settings<configuration-backends>` are selected. For our example, the backend configuration would be:
+
+.. code-block:: python
+
+    kinto.storage_backend = kinto.core.storage.postgresql
+    kinto.storage_url = postgres://dbuser:dbpassword@localhost/dbname
+    kinto.cache_backend = kinto.core.cache.postgresql
+    kinto.cache_url = postgres://dbuser:dbpassword@localhost/dbname
+    kinto.permission_backend = kinto.core.permission.postgresql
+    kinto.permission_url = postgres://dbuser:dbpassword@localhost/dbname
+
+
+Initialization
+--------------
 
 The last step consists in creating the necessary tables and indices, run the ``migrate`` command:
 
-.. code-block :: bash
+.. code-block:: bash
 
     $ kinto --ini production.ini migrate
 
```

Comment:
We could add a `..important::` paragraph here to mention that this command should be ran every time Kinto is upgraded.